CMakeLists.txt

```diff
@@ -161,6 +161,8 @@ install(
         GROUP_EXECUTE
         WORLD_READ
         WORLD_EXECUTE
+    RENAME
+        convert-hf-to-gguf.py
```

Comment:
It's better to switch to snake case everywhere for the Python scripts, so probably no need to rename here?

Reply:
Absolutely. Assuming backwards compatibility is not a concern, removed it.